cfy_manager/components/base_component/base_component.py

```diff
@@ -82,3 +88,90 @@ def validate_dependencies(self):
             self.logger.debug(
                 'All prerequisites for {class_name} are met'.format(
                     class_name=self.__class__.__name__))
+
+    def use_supplied_certificates(self, cert_destination, key_destination,
+                                  ca_destination,
+                                  owner=CLOUDIFY_USER, group=CLOUDIFY_GROUP):
+        """
+            Use user-supplied certificates, after checking they're not
```

Comment:
Indentation not needed in the docstring

Reply:
But it reads much more nicely <smiles>

Reply:
But it's inconsistent with the rest of our docstring :D

Reply:
Not all of them- just the ones where people didn't notice I'd done it!
But yes, fair point, will change.